.env.dev.sample

```diff
@@ -1,3 +1,4 @@
 ; To use dev env ini file.
 ; cp .env.dev.sample .env
-DEBUG=on
\ No newline at end of file
+DEBUG=on
+DATABASE_URL=sqlite:///anem-per-feina.db
```

Comment:
Maybe it's more clear if we preserve the `.sqlite` extension. I know that `.db` also is normally interpreted as sqlite, but maybe it's simpler and avoids misunderstandings 🤷🏻‍♂